### PR TITLE
Update application description with latest information

### PR DESCRIPTION
**Pull request by sgchoi5**
- Add new C# sample
- Add new code link
- Add <a> tag for doc link
- Add <code> tag for some class

### CODE REVIEW

**Comment by sgchoi5** [comment]
now.. checking output..

**Comment by sgchoi5** [comment]
Checking... done..
Wait for review... : )   

TizenDocsBot does not generate new test page... last https://docs.stg.tizen.org/staging/678 has old content.. hm...

File: docs/application/dotnet/samples/ui/overview.md [hunk 1]
```diff
@@ -386,7 +395,7 @@ for TD:
 			<p><img alt="" height="225" src="media/tv36cubewithskiasharp.png" width="400" /></p>
 			</td>
 			<td>
-			<p><strong>(T) CubeWithSkiaSharp (OpenTK)</strong> [In progress]</p>
+                        <p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/TV/CubeWithSkiaSharp" target="_blank"><strong>(T) CubeWithSkiaSharp (OpenTK)</strong></a></p>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/ui/overview.md]: Fixed the README page of repo

File: docs/application/dotnet/samples/ui/overview.md [hunk 2]
```diff
@@ -164,11 +164,20 @@ for TD:
 			</ul></p>
 			</td>
 		</tr>
+		<tr>
+			<td>
+			<p><img alt="" height="180" src="media/m28calendarcomponent.png" width="180"/></p>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/ui/overview.md]: At now, it displays properly. plz, check it again.
https://docs.stg.tizen.org/staging/678/application/dotnet/samples/social/overview

File: docs/application/dotnet/samples/network/overview.md [hunk 3]
```diff
@@ -33,29 +33,29 @@ for TD:
 			<td><img alt="" height="267" src="media/m17networkapp.png" width="150"/></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/NetworkApp" target="_blank"><strong>(M) NetworkApp</strong></a></p>
-			<p>This sample application demonstrates how to use <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Network.Connection.html" target="_blank">Tizen.Network.Connection C# API</a>, Wi-Fi, and Wi-Fi Direct.<br />
+			<p>This sample application demonstrates how to use <a href="https://samsung.github.io/TizenFX/API5/api/Tizen.Network.Connection.html" target="_blank">Tizen.Network.Conncetion</a>, Wi-Fi, and Wi-Fi Direct.<br />
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/network/overview.md]: Fixed

File: docs/application/dotnet/samples/network/overview.md [hunk 4]
```diff
@@ -69,21 +69,21 @@ for TD:
 			<td><img alt="" height="180" src="media/wlescanner.png" width="180"/></td>
 			<td>
                         <p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/Lescanner" target="_blank"><strong>(W) Lescanner</strong></a></p>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/network/overview.md]: I am waiting for merging code.
If it requires some more time, I will change it to "In progress" again

File: docs/application/dotnet/samples/general/overview.md [hunk 5]
```diff
@@ -26,7 +26,7 @@ for TD:
 			<td><img alt="" height="267" src="media/m37weather.png" width="150"/></td>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/general/overview.md]: Fixed

File: docs/application/dotnet/samples/ui/overview.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/dotnet/samples/ui/overview.md]: I checked the link but we don't have image in that link neither. The code says it is using the image under "dotnet/samples/ui/media", but we don't have image with name "m28calendarcomponent.png" under there.
Also, I'm not sure the image "**calendarcomponent**" is used for **TextReader**.
Would you please check it once more?

File: docs/application/dotnet/samples/general/overview.md [hunk 6]
```diff
@@ -40,8 +40,8 @@ for TD:
 			<td><img alt="" height="180" src="media/wplayingwithhwInputs.png" width="180"/></td>
 			<td>
                         <p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/PlayingWithHWInputs" target="_blank"><strong>(W) PlayingWithHWInputs</strong></a></p>
-			<p>This sample application demonstrates how to handle rotary events from bezel of a wearable device and how to create circular UI using <a href="https://github.com/Samsung/Tizen.CircularUI/" target="_blank">Tizen.Wearable.CircularUI C# API</a>. In addition, it shows the way how to detect when the back button is pressed.</p>
-			<p>This application uses Tizen.Wearable.CircularUI API.<br>
+			<p>This sample application demonstrates how to handle rotary events from bezel of a wearable device and how to create circular UI using <a href="https://samsung.github.io/Tizen.CircularUI/api/index.html" target="_blank">Tizen.Wearable.CircularUI</a>. In addition, it shows the way how to detect when the back button is pressed.</p>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/general/overview.md]: Gear device has the rotating bezel. This sample is using this feature and try to explain how to use it.

File: docs/application/dotnet/samples/appfw/overview.md [hunk 7]
```diff
@@ -162,13 +162,12 @@ for TD:
 			You can install it by using Visual Studio or the sdb command line as follows:<br>
 			<code>$ sdb install org.tizen.example.ServiceApp-1.0.0.tpk</code><br>
 			<code>$ sdb shell</code><br>
-			<code>$ app_launcher -s org.tizen.example.ServiceApp
-			</code><br>
+			<code>$ app_launcher -s org.tizen.example.ServiceApp</code><br>
 			If you do not install Tizen .Net service application, an exception occurs when you press <strong>Send a message</strong> button.
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/appfw/overview.md]: Fixed

File: docs/application/dotnet/samples/account/overview.md [hunk 8]
```diff
@@ -54,7 +54,7 @@ for TD:
 			<td><img alt="" height="180" src="media/waccounts.png" width="180"/></td>
 			<td>
                         <p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/Accounts" target="_blank"><strong>(W) Accounts</strong></a></p>
-			<p>This sample application demonstrates how to implement an account provider, which can add and configure an account. It also demonstrates how to create circular UI using <a href="https://github.com/Samsung/Tizen.CircularUI/" target="_blank">Tizen.Wearable.CircularUI C# API</a>.</p>
+			<p>This sample application demonstrates how to implement an account provider, which can add and configure an account using <a href="https://samsung.github.io/TizenFX/API5/api/Tizen.Account.AccountManager.html" target="_blank">Tizen.Account.AccountManager</a>. It also demonstrates how to create circular UI using <a href="https://samsung.github.io/Tizen.CircularUI/api/index.html" target="_blank">Tizen.Wearable.CircularUI</a>.</p>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/account/overview.md]: Fixed

File: docs/application/dotnet/samples/location/overview.md [hunk 9]
```diff
@@ -26,7 +26,7 @@ for TD:
 			<td><img alt="" height="267" src="media/m13geofence.png" width="150"/></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/Geofence" target="_blank"><strong>(M) Geofence</strong></a></p>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/location/overview.md]: I've sent a request to app developer. will be updated.

File: docs/application/dotnet/samples/messaging/overview.md [hunk 10]
```diff
@@ -26,7 +26,7 @@ for TD:
 			<td><img alt="" height="267" src="media/m19pushreceiver.png" width="150"/></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/PushReceiver" target="_blank"><strong>(M) PushReceiver</strong></a></p>
-			<p>This sample application demonstrates how to receive push notification from the application server using the push server.</p>
+			<p>This sample application demonstrates how to receive push notification from the application server using the push server and <a href="https://samsung.github.io/TizenFX/API5/api/Tizen.Messaging.Push.html" target="_blank">Tizen.Messaging.Push</a></p>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/messaging/overview.md]: Fixed.

File: docs/application/dotnet/samples/watch/overview.md [hunk 11]
```diff
@@ -36,14 +36,13 @@ for TD:
...
 			</p>
 			<p>In <code>tizen-manifest.xml</code> file, set <code>ambient-support</code> attribute to true:<br>
 			For more information, see ambient-support attribute.
 			<br>
-			<code>
-			&lt;watch-application appid&equals;&quot;org.tizen.example.AmbientWatch&quot;  exec&equals;&quot;AmbientWatch.dll&quot; type&equals;&quot;dotnet&quot;  ambient-support&equals;&quot;true&quot;&gt;</code>
+			<code>&lt;watch-application appid&equals;&quot;org.tizen.example.AmbientWatch&quot;  exec&equals;&quot;AmbientWatch.dll&quot; type&equals;&quot;dotnet&quot;  ambient-support&equals;&quot;true&quot;&gt;</code>
 			</p>
 			<p>
 			You can get details about <a href="../../../tizen-studio/native-tools/manifest-text-editor.md" target="_blank">ambient-support attribute</a>.
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/watch/overview.md]: Added link.

File: docs/application/dotnet/samples/multimedia/overview.md [hunk 12]
```diff
@@ -25,43 +25,43 @@ for TD:
...
+                        <p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/AudioIOSample" target="_blank"><strong>(M) AudioIOSample</strong></a></p>
+			<p>This sample application demonstrates how to capture an audio and play it using <a href="https://samsung.github.io/TizenFX/API5/api/Tizen.Multimedia.html"  target="_blank">Tizen.Multimedia</a>.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="267" src="media/m41camera.png" width="150"/></td>
 			<td>
                         <p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/Camera" target="_blank"><strong>(M) Camera</strong></a></p>
-			<p>This sample application demonstrates how to take a picture using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.html" target="_blank">Tizen.Multimedia C# API</a>.</p>
+			<p>This sample application demonstrates how to take a picture using <a href="https://samsung.github.io/TizenFX/API5/api/Tizen.Multimedia.html" target="_blank">Tizen.Multimedia</a>.</p>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/multimedia/overview.md]: changed image

File: docs/application/dotnet/samples/context/overview.md [hunk 13]
```diff
@@ -40,14 +40,15 @@ for TD:
 			<td><img alt="" height="180" src="media/wfrequentlyused.png" width="180"/></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/AppHistory" target="_blank"><strong>(W) AppHistory</strong></a></p>
-			<p>This sample application demonstrates how to retrieve your application usage patterns. This sample uses XAML files for GUI.</p>
+			<p>This sample application demonstrates how to retrieve your application usage patterns using <a href="https://samsung.github.io/TizenFX/API5/api/Tizen.Context.AppHistory.html" target="_blank">Tizen.Context.AppHistory</a>.
+                        <p>This sample uses XAML files for GUI.</p>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/context/overview.md]: Fixed.

File: docs/application/dotnet/samples/appfw/overview.md [hunk 14]
```diff
@@ -95,16 +95,16 @@ for TD:
 			<td><img alt="" height="180" src="media/wappcontrol.png" width="180"/></td>
 			<td>
                         <p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/AppControl" target="_blank"><strong>(W) AppControl</strong></a></p>
-			<p>This sample application demonstrates how to launch an application, get a result for the launch request, and create circular UI using <a href="https://github.com/Samsung/Tizen.CircularUI/" target="_blank">Tizen.Wearable.CircularUI C# API</a>. When you press <strong>Launch</strong> button, AppInformation sample application will be launched.</p>
+			<p>This sample application demonstrates how to launch an application, get a result for the launch request, and create circular UI using <a href="https://samsung.github.io/Tizen.CircularUI/api/index.html" target="_blank">Tizen.Wearable.CircularUI</a>. When you press <strong>Launch</strong> button, AppInformation sample application will be launched.</p>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/appfw/overview.md]: Fixed.

File: docs/application/dotnet/samples/appfw/overview.md [hunk 15]
```diff
@@ -26,14 +26,14 @@ for TD:
 			<td><img alt="" height="267" src="media/m40alarms.png" width="150"/></td>
 			<td>
 			<p><strong>(M) Alarms</strong> [In progress]</p>
-			<p>This sample application demonstrates how to schedule start of application at specified date and time or after delay using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.html" target="_blank">Tizen.Applications C# API</a>.</p>
+			<p>This sample application demonstrates how to schedule start of application at specified date and time or after delay using <a href="https://samsung.github.io/TizenFX/API5/api/Tizen.Applications.html" target="_blank">Tizen.Applications</a>.</p>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/appfw/overview.md]: Fixed.

File: docs/application/dotnet/samples/ui/overview.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/ui/overview.md]: Sorry, It was my mistake. I've changed proper image for TextReader.

File: docs/application/dotnet/samples/general/overview.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (surbhig12) [reply docs/application/dotnet/samples/general/overview.md]: Suggest rephrasing to:

This sample application demonstrates how to handle rotary events of the bezel of a wearable device and how to create circular UI using....

Please provide more information for the following sentence:
In addition, it shows the way how to detect when the back button is pressed.

Can we rephrase it to the following and does it convey the same meaning:
In addition, it shows how to detect the navigation path when the back button is pressed.

File: docs/application/dotnet/samples/uix/overview.md [hunk 16]
```diff
@@ -44,7 +44,15 @@ for TD:
...
 	<tbody>
 		<tr>
-			<td>There is no sample. </td>
+			<tr>
+			<td>
+			<p><img alt="" height="180" src="media/w67textreader.png" width="180"/></p>
+			</td>
+			<td>
+			<p><strong>(W) TextReader</strong> [In progress]</p>
+			<p>This sample application demonstrates how to transform input text into sound data using <a href="https://samsung.github.io/TizenFX/API5/api/Tizen.Uix.Tts.html">Tizen.Uix.Tts</a>.</p>
```
Comment (mijin85cho) [inline docs/application/dotnet/samples/uix/overview.md]: @surbhig12 This TextReader for wearable is no included in this page. As we have TextReader for Mobile and TV already in this page, how about using the same sentence for all these 3 TextReader apps?

Comment (surbhig12) [reply docs/application/dotnet/samples/uix/overview.md]: Hello @mijin85cho,

Please correct me if I am wrong, I saw all the sentences again and the format seems the same:
This sample application demonstrates how to.......... using......
I am not sure if I got your suggestion right.

Comment (mijin85cho) [reply docs/application/dotnet/samples/uix/overview.md]: Hello, @surbhig12 

For now, the descriptions are
 - **TextReader for Mobile**
   This sample application demonstrates how to synthesize voice from text using Tizen.Uix.Tts.
 - **TextReader for Wearable**
   This sample application demonstrates how to transform input text into sound data using Tizen.Uix.Tts.
 - **TextReader for TV**
   This sample application demonstrates how to synthesize voice from text using Tizen.Uix.Tts.

TextReader for Wearable and TextReader for Mobile & TV are using different sentences.
I'm just suggesting to use a same description to these TextReader, if they are describing same function for Mobile, Wearable and TV.

Comment (surbhig12) [reply docs/application/dotnet/samples/uix/overview.md]: @mijin85cho,
Thank you for clarifying. I agree with your point. We can use the same description.

File: docs/application/dotnet/samples/multimedia/overview.md [hunk 17]
```diff
@@ -25,43 +25,43 @@ for TD:
...
 		<tr>
 			<td><img alt="" height="267" src="media/m51radiosample.png" width="150"/></td>
 			<td>
-			<p><strong>(M) RadioSample</strong> [In progress]</p>
-			<p>This sample application demonstrates how to play the radio using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.html"  target="_blank">Tizen.Multimedia C# API</a>.</p>
+                        <p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/RadioSample" target="_blank"><strong>(M) RadioSample</strong></a></p>
+			<p>This sample application demonstrates how to play the radio using <a href="https://samsung.github.io/TizenFX/API5/api/Tizen.Multimedia.html"  target="_blank">Tizen.Multimedia</a>.</p>
 			</td>
 		</tr>
 		<tr>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/multimedia/overview.md]: 'SINK' is generally used term in Screen Mirroring for client side.

https://developer.tizen.org/ko/development/api-references/native-application?redirect=https://developer.tizen.org/dev-guide/5.0.0/org.tizen.native.mobile.apireference/group__CAPI__MEDIA__SCREEN__MIRRORING__MODULE.html

File: docs/application/dotnet/samples/uix/overview.md (hunk 16 above)
Comment (sgchoi5) [reply docs/application/dotnet/samples/uix/overview.md]: Fixed to use same description...

File: docs/application/dotnet/samples/appfw/overview.md [hunk 18]
```diff
@@ -180,17 +179,17 @@ for TD:
 			<td><img alt="" height="180" src="media/wpreference.png" width="180"/></td>
 			<td>
                         <p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/Preference" target="_blank"><strong>(W) Preference</strong></a></p>
-			<p>This sample application demonstrates how to use <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.Preference.html" target="_blank">Tizen.Applications.Preference C# API</a> and how to create circular UI using <a href="https://github.com/Samsung/Tizen.CircularUI/" target="_blank">Tizen.Wearable.CircularUI C# API</a>.</p>
+			<p>This sample application demonstrates how to use <a href="https://samsung.github.io/TizenFX/API5/api/Tizen.Applications.Preference.html" target="_blank">Tizen.Applications.Preference</a> and how to create circular UI using <a href="https://samsung.github.io/Tizen.CircularUI/api/index.html" target="_blank">Tizen.Wearable.CircularUI</a>.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="180" src="media/wserviceapp.png" width="180"/></td>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/appfw/overview.md]: Fixed

File: docs/application/dotnet/samples/watch/overview.md [hunk 19]
```diff
@@ -36,14 +36,13 @@ for TD:
...
+			<p>This sample application demonstrates how to create a watch application that supports ambient mode in Tizen wearable device. It also demonstrates how to create a circular watch face using <a href="https://samsung.github.io/Tizen.CircularUI/api/index.html" target="_blank">Tizen.Wearable.CircularUI</a>.<br>
 			To extend battery life, a watch application can display a limited UI in the ambient mode by detecting ambient mode changes.<br>
 			</p>
 			<p>In <code>tizen-manifest.xml</code> file, set <code>ambient-support</code> attribute to true:<br>
-			For more information, see ambient-support attribute.
+			For more information, see <a href="../../../tizen-studio/native-tools/manifest-text-editor.md" target="_blank">ambient-support attribute</a>.
 			<br>
-			<code>
-			&lt;watch-application appid&equals;&quot;org.tizen.example.AmbientWatch&quot;  exec&equals;&quot;AmbientWatch.dll&quot; type&equals;&quot;dotnet&quot;  ambient-support&equals;&quot;true&quot;&gt;</code>
+			<code><watch-application ambient-support="true" appid="org.tizen.watchsample" exec="watchsample" setup-appid="watchface-editor"></code>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/watch/overview.md]: Fixed

[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/dotnet/samples/watch/overview.md]: Just for Checking. The latest for now is not the updated version, as it is reverted to original. 
Would you please confirm once more that you'd like to use original?

- Original & the latest : 
   \<watch-application appid="org.tizen.example.AmbientWatch" exec="AmbientWatch.dll" type="dotnet" ambient-support="true"\>

- Previous version : 
   \<watch-application ambient-support="true" appid="org.tizen.watchsample" exec="watchsample" setup-appid="watchface-editor"\>

File: docs/application/dotnet/samples/multimedia/overview.md (hunk 17 above)
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/dotnet/samples/multimedia/overview.md]: Thanks for the answer.

**Comment by sgchoi5** [comment]
I've made Watchface application by using Visual Studio plugin.
From manifest file, it will be used.
<watch-application appid="org.tizen.example.TizenWatchfaceApp1" exec="TizenWatchfaceApp1.dll" type="dotnet" ambient-support="true">

&lt;watch-application appid=&quot;org.tizen.example.TizenWatchfaceApp1&quot; exec=&quot;TizenWatchfaceApp1.dll&quot; type=&quot;dotnet&quot; ambient-support=&quot;true&quot;&gt;

**Comment by mijin85cho** [comment]
Thanks for the updates, @sgchoi5.

@surbhig12 Would you please review this PR again? sgchoi5 updated it.

File: docs/application/dotnet/samples/general/overview.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (surbhig12) [reply docs/application/dotnet/samples/general/overview.md]: Hello, please clarify this doubt.

File: docs/application/dotnet/samples/location/overview.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (surbhig12) [reply docs/application/dotnet/samples/location/overview.md]: Same comment. 

Suggestion:
Please add Readme.md files for the Mobile samples

File: docs/application/dotnet/samples/appfw/overview.md [hunk 20]
```diff
@@ -117,11 +117,11 @@ for TD:
 			<td><img alt="" height="180" src="media/wappinformation.png" width="180"/></td>
 			<td>
                         <p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/AppInformation" target="_blank"><strong>(W) AppInformation</strong></a></p>
-			<p>This sample application demonstrates how to get the specific information such as AppId, Package ID, and shared directory path. It also demonstrates how to get the directory information such as resource, cache, app data and how to create circular UI using <a href="https://github.com/Samsung/Tizen.CircularUI/" target="_blank">Tizen.Wearable.CircularUI C# API</a>.</p>
-			<p>This application uses Tizen.Application API.<br>
+			<p>This sample application demonstrates how to get the specific information such as AppId, Package ID, and shared directory path. It also demonstrates how to get the directory information such as resource, cache, app data and how to create circular UI using <a href="https://samsung.github.io/Tizen.CircularUI/api/index.html" target="_blank">Tizen.Wearable.CircularUI</a>.</p>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/appfw/overview.md]: Fixed

**Comment by sgchoi5** [comment]
I've asked Sample app developer to add README page again.
PlayingWithHWInputs  =>  Gear device has the rotating bezel (hardware).
It demonstrates how to use it.

File: docs/application/dotnet/samples/general/overview.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (surbhig12) [reply docs/application/dotnet/samples/general/overview.md]: @sgchoi5,
Thank you for the response.
Please let me know if it would it be okay to rephrase the last sentence as follows:

In addition, it demonstrates how to use the rotating bezel.

**Comment by sgchoi5** [comment]
@surbhig12 Updated!

**Comment by mijin85cho** [comment]
@surbhig12 When the author updated the PR according to your comment, please check it. We are waiting your response for this PR.

**Comment by surbhig12** [comment]
> @surbhig12 When the author updated the PR according to your comment, please check it. We are waiting your response for this PR.

@mijin85cho, I have reviewed all the files again. However the readme files for docs/application/dotnet/samples/location/overview.md are still not added. I was waiting for sometime to let the author update. I am fine with all the other changes.
